packages/frontend/package.json

```
@@ -61,7 +61,7 @@
     "@vitejs/plugin-react-refresh": "1.3.6",
     "react-bootstrap-icons": "1.3.0",
     "typescript": "~5.4.5",
-    "vite": "5.2.14"
+    "vite": "6.0.0"
```

nit: use the latest at the time of code change?
```suggestion
    "vite": "6.0.7"
```